catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ResourceDownloadEndpoint.java

```diff
@@ -93,12 +174,25 @@ public Response startDownloadToCacheOnly(@PathParam("sourceId") String sourceId,
             LOGGER.error(message, e);
             throw new DownloadToCacheOnlyException(Status.NOT_FOUND, message);
         }
+    }
 
-        String message = String.format(
-                "The product associated with metacard [%s] from source [%s] is being downloaded to the product cache.",
-                metacardId,
-                sourceId);
-        return Response.ok(message, MediaType.TEXT_PLAIN_TYPE)
-                .build();
+    /**
+     * Gets the list of active product downloads.
+     *
+     * @return response object containing the list of all the currently active downloads
+     * @throws DownloadToCacheOnlyException thrown if the product download couldn't be started
+     */
+    @GET
+    @Produces(APPLICATION_JSON)
+    public Response getDownloadList() throws DownloadToCacheOnlyException {
+        try {
+            List<DownloadInfo> downloadsInProgress = downloadManager.getDownloadsInProgress();
+            return Response.ok(objectMapper.toJson(downloadsInProgress))
+                    .build();
+        } catch (RuntimeException e) {
+            LOGGER.error("Unable to get list of downloads.", e);
+            throw new DownloadToCacheOnlyException(Status.INTERNAL_SERVER_ERROR,
+                    "Unable to get list of downloads");
```

Comment:
We shouldn't be passing up e as the cause to any external client, which is where this throw seems to go.